src/components/Badges.tsx

```
@@ -0,0 +1,18 @@
+interface BadgeProps {
+  color: string; 
+  text: string;
+  icon: string; 
+}
+
+export default function Badges({
+  color,
+  text,
+  icon
```

ESLint

[error] 'icon' is defined but never used.

(@typescript-eslint/no-unused-vars)